solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClient.java

```diff
@@ -813,6 +813,7 @@ public String getBaseURL() {
    * In this case the client is more flexible and can be used to send requests to any cores. The
    * cost of this is that the core must be specified on each request.
    */
+  @Deprecated
```

Comment:
Eventually probably want to have a `use this instead` part of the javadoc

Reply:
Just scavenged some text from a patch file that @gerlowskija had written!